### PR TITLE
10.post/api/reviews/b yreview id/comments

### PR DESCRIPTION
**Pull request by davegreensmith**
add endpoint and tests for posting comments by review_id

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -334,3 +334,82 @@ describe('GET /api/reviews/:review_id/comments', () => {
...
+        .post('/api/reviews/1/comments')
+        .send()
+        .expect(400)
+        .then(({ body: { msg } }) => {
+          expect(msg).toBe('Bad request');
+        });
+    });
+    test('400: Bad request response where username does not exist in users table', () => {
+      const postComment = {
+        username: 'salehTheGreat',
```
Comment (two-triangles) [inline __tests__/app.test.js:403]: oh you :p

File: __tests__/app.test.js [hunk 2]
```diff
@@ -334,3 +334,82 @@ describe('GET /api/reviews/:review_id/comments', () => {
...
+        body: 'Padme loves the box it came in',
```
Comment (two-triangles) [inline __tests__/app.test.js:404]: * she :P

File: __tests__/app.test.js [hunk 3]
```diff
@@ -334,3 +334,82 @@ describe('GET /api/reviews/:review_id/comments', () => {
...
+      const postComment = {};
+      return request(app)
+        .post('/api/reviews/1/comments')
+        .send()
+        .expect(400)
+        .then(({ body: { msg } }) => {
+          expect(msg).toBe('Bad request');
+        });
+    });
+    test('400: Bad request response where username does not exist in users table', () => {
```
Comment (two-triangles) [inline __tests__/app.test.js:401]: I think this would fall under a 404 - as the user salehTheGreat could exist at some point :)

**Comment by two-triangles** [comment]
Have a look over the comments :) Some good stuff but a few things to change

**Review by emilyb93 — APPROVED** [review]

**Pull request merged**